aws/table_aws_wellarchitected_consolidated_report.go

```diff
@@ -0,0 +1,156 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+
+	wellarchitectedv1 "github.com/aws/aws-sdk-go/service/wellarchitected"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWellArchitectedConsolidatedReport(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wellarchitected_consolidated_report",
+		Description: "AWS Well-Architected Consolidated Report",
+		List: &plugin.ListConfig{
+			Hydrate: listWellArchitectedConsolidatedReports,
+			KeyColumns: []*plugin.KeyColumn{
```

Comment:
@ParthaI Does the API return different results if the `IncludeSharedResources` req param is set to `true` or `false`? What’s the default for this param?